app/templates/base.html

```diff
@@ -111,7 +111,9 @@ <h2 class="govuk-visually-hidden">Support links</h2>
 {% endblock %}
 {% block bodyEnd %}
     <!--[if gt IE 8]><!-->
-    <script type="module" src="{{ url_for('static', filename='govuk-frontend.min.js') }}">
+    <script type="module"
+            src="{{ url_for('static', filename='govuk-frontend.min.js') }}"></script>
```

Comment:
why do we need this separate script block? Does the first one import everything from that js file and the second just initalAll and call it? How do we use the stuff from the first one and how does it fix the bug?

Reply:
@anthonyhashemi they had something like this in the documentation while is why I've added that line but I've removed it and it seems to work just fine
<img width="784" alt="image" src="https://github.com/user-attachments/assets/46828d58-999d-456a-8232-801e1c83c2d6">

Reply:
interesting, okay, if it works without sounds good